app/components/post_textbox/post_textbox_base.js

```diff
@@ -593,6 +614,14 @@ export default class PostTextBoxBase extends PureComponent {
         return (/\B@(all|channel)\b/i).test(textWithoutCode);
     };
 
+    groupsMentionedInText = (text) => {
+        const {groupsWithAllowReference} = this.props;
+        const AT_MENTION_PATTERN = /\B@([a-z0-9.\-_]+)/gi;
+        const textWithoutCode = text.replace(/(`+)([^`]|[^`][\s\S]*?[^`])\1(?!`)| *(`{3,}|~{3,})[ .]*(\S+)? *\n([\s\S]*?\s*)\3 *(?:\n+|$)/g, '');
```

Comment:
might be worth adding this to the constants file too

Reply:
Done!

Reply:
@fmunshi 
Actually turned out that the AT_MENTION_REGEX doesn't work for my case. I needed to remove the `$` and also add a `g` flag cause it only picked up one at mention in the string of at mentions.